src/css/components/panel.less

```diff
@@ -8,7 +8,7 @@
 
 .panel-group .panel-inverse {
   border-radius: 0;
-  border-top: solid 1px #3d4045;
+  border-top: solid 1px #3c4457;
```

Comment:
This is the same value as `@navbar-border-color` maybe we should use the variable.